test/functional/editions_controller_test.rb

```diff
@@ -264,6 +264,46 @@ class EditionsControllerTest < ActionController::TestCase
     end
   end
 
+  context "#duplicate" do
+    setup do
+      @answer = FactoryBot.create(:answer_edition, panopticon_id: FactoryBot.create(:artefact).id)
+    end
+
+    should "redirect to edit tab and show success message when user saves successfully" do
+      EditionDuplicator.any_instance.expects(:duplicate).returns(true)
+      EditionDuplicator.any_instance.expects(:new_edition).returns(@answer)
+      post :duplicate, params: { id: @edition.id, to: "help_page" }
```

Comment:
I think it's a bit confusing for the reader that we're posting a request to create a help page, but stubbing the `EditionDuplicator` to return an answer.

Reply:
> I think it's a bit confusing for the reader that we're posting a request to create a help page, but stubbing the `EditionDuplicator` to return an answer.

Good point. I think I've sorted that by creating a new (help) edition and stubbing that as the expected return.